opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/snmp/IpAddressTable.java

```diff
@@ -235,8 +237,7 @@ public void updateIpInterfaceData(final OnmsNode node, final String ipAddr) {
             ipIf.setSnmpInterface(snmpIf);
 
         }
-
-        ipIf.setIpHostName(ipAddr);
```

Comment:
This is one of the place where existing hostname in ipInterface is getting overwritten